### PR TITLE
tftputil-gui: Add version 1.4.5

### PR DESCRIPTION
**Pull request by issaclin32**
**TFTPUtil GUI** ([homepage](https://sourceforge.net/projects/tftputil/)) is a TFTP server and client suite written in .NET framework.

Notes:
* The `0BSD` license ([BSD Zero-Clause license](https://spdx.org/licenses/0BSD.html)) is contained in the archive.
* TFTPUtil has not been updated since 2014. Therefore *checkver* and *autoupdate* is not needed.

### CODE REVIEW

File: bucket/tftputil-gui.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.4.5",
+    "description": "TFTP server and client written in .NET framework.",
+    "homepage": "https://sourceforge.net/projects/tftputil/",
+    "license": "0BSD",
+    "url": "https://dl.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%201.4.5/TFTPUtil_GUI_Version_1.4.5_Binary_Installer.exe#/dl.7z",
+    "hash": "sha1:4f7ecfd0abdd2f7c48438ce64e22981fba6daff4",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\src\", \"$dir\\uninstall.exe\" -Force -Recurse",
+    "bin": [
```
Comment (Ash258) [inline bucket/tftputil-gui.json]: Do you think these are good and useful binaries?

File: bucket/tftputil-gui.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            "TFTPUtil GUI"
+        ],
+        [
+            "TFTPUtil Client GUI.exe",
+            "TFTPUtil Client GUI"
+        ]
+    ],
+    "checkver": "TFTPUtil_GUI_Version_([\\d.]+)",
+    "autoupdate": {
+        "url": "https://dl.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%20$version/TFTPUtil_GUI_Version_$version_Binary_Installer.exe#/dl.7z"
```
Comment (Ash258) [inline bucket/tftputil-gui.json]: Why did you select the slowest mirror, which is not working most of the time for wide variety of countries?

```suggestion
        "url": "https://downloads.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%20$version/TFTPUtil_GUI_Version_$version_Binary_Installer.exe"
```

image

File: bucket/tftputil-gui.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
+{
+    "version": "1.4.5",
+    "description": "TFTP server and client written in .NET framework.",
```
Comment (Ash258) [inline bucket/tftputil-gui.json]: Why is source code language relevant for end user?

File: bucket/tftputil-gui.json [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    "homepage": "https://sourceforge.net/projects/tftputil/",
+    "license": "0BSD",
+    "url": "https://dl.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%201.4.5/TFTPUtil_GUI_Version_1.4.5_Binary_Installer.exe#/dl.7z",
+    "hash": "sha1:4f7ecfd0abdd2f7c48438ce64e22981fba6daff4",
+    "post_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\src\", \"$dir\\uninstall.exe\" -Force -Recurse",
```
Comment (Ash258) [inline bucket/tftputil-gui.json]:
```suggestion
    "pre_install": "Remove-Item \"$dir\\`$PLUGINSDIR\", \"$dir\\src\", \"$dir\\uninstall.exe\" -Force -Recurse",
```

File: bucket/tftputil-gui.json (hunk 2 above)
Comment (issaclin32) [reply bucket/tftputil-gui.json]: Didn't know that. Thanks for pointing that out.

File: bucket/tftputil-gui.json (hunk 1 above)
Comment (Ash258) [reply bucket/tftputil-gui.json]: Remove binaries. They do not have any parameter handling.

File: bucket/tftputil-gui.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
+{
+    "version": "1.4.5",
+    "description": "TFTP server and client.",
+    "homepage": "https://sourceforge.net/projects/tftputil/",
+    "license": "0BSD",
+    "url": "https://downloads.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%201.4.5/TFTPUtil_GUI_Version_1.4.5_Binary_Installer.exe",
```
Comment (Ash258) [inline bucket/tftputil-gui.json]:
```suggestion
    "url": "https://downloads.sourceforge.net/project/tftputil/TFTPUtil/TFTPUtil%20Version%201.4.5/TFTPUtil_GUI_Version_1.4.5_Binary_Installer.exe#/dl.7z",
```

File: bucket/tftputil-gui.json (hunk 1 above)
Comment (issaclin32) [reply bucket/tftputil-gui.json]: > Why do you not test installation/uninstallation before posting PR?

Sorry about that. I will pay attention when I apply suggestions.

**Comment by issaclin32** [comment]
> Why do you not test installation/uninstallation before posting PR?

Sorry about that. I will pay attention when I apply suggestions.

File: bucket/tftputil-gui.json [hunk 6]
```diff
@@ -0,0 +1,19 @@
+{
+    "version": "1.4.5",
+    "description": "TFTP server and client.",
```
Comment (Ash258) [inline bucket/tftputil-gui.json]:
```suggestion
    "description": "TFTP server and client",
```

**Pull request merged**